cmd/tracee-ebpf/internal/flags/flags-output.go

```diff
@@ -29,6 +30,7 @@ option:{stack-addresses,detect-syscall,exec-env,relative-time,exec-hash,parse-ar
   exec-hash                                        when tracing sched_process_exec, show the file hash(sha256) and ctime
   parse-arguments                                  do not show raw machine-readable values for event arguments, instead parse into human readable strings
   sort-events                                      enable sorting events before passing to them output. This will decrease the overall program efficiency.
+  protocol                                         write output in compatible format for tracee-rules (relevant only when sending as json or gob)
```

Comment:
ugly-ish json or gob only limitation... thinking if we can do something better...

Reply:
It wouldn't be relevant in any other case (in Open Source), since messaging is done only in those formats. I understand not liking the lack of genericity but so far I haven't come with a better idea (without actually using generics which don't yet exist).

Reply:
I've removed it as an option in a new commit (`Print` now takes a `protocol.Event`), please take a look.